tests/Functional/Ticket/PR6031Test.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Sonata\AdminBundle\Tests\Functional\WebTestCase;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+final class PR6031Test extends WebTestCase
+{
+    public function testLabelInShowAction(): void
```

Comment:
Since this test is not performing any direct assertion against the label, we should add a description explaining how it's related.